target-appTests/HomeViewModelTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  HomeViewModelTests.swift
+//  target-appTests
+//
+//  Created by Andres Leonel Torrico Cossio on 17/10/2022.
+//
+
+import XCTest
+import Combine
+@testable import target_app
+
+class HomeViewModelTests: XCTestCase {
+    
+    var viewModel: HomeViewModel!
+    private var cancellables: Set<AnyCancellable>!
+
+    override func setUp() {
+        super.setUp()
+        viewModel = HomeViewModel(
+            targetServices: TargetServicesMock()
+        )
+        cancellables = []
+    }
+
+    func testGetTargetsSuccessfull() {
+        let expect = expectation(description: "get targets successfull")
+        
+        viewModel.$targets
+            .dropFirst()
```

Comment:
Instead of dropping the first value we could explicitly filter values we are not interested in, so the test is more clear

Reply:
Excellent! Done!

I also had to conform Equatable protocol in my model class to add the filter: `.filter { $0 != [] }`

```
extension Target: Equatable {}

func ==(lhs: Target, rhs: Target) -> Bool {
    let areEqual = lhs.id == rhs.id

    return areEqual
}
```